### PR TITLE
Flux TICKscript package

### LINKED ISSUE #2346: Flux TICKscript package
**PR:** https://github.com/influxdata/flux/pull/3427

The new `contrib/bonitoo/tickscript` package provides helper functions to make it easier to migrate Kapacitor TICKscripts into Flux scripts.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2346

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+related:
+  - /{{< latest "kapacitor" >}}/nodes/alert_node/, Kapacitor AlertNode
+---
+
+The `tickscript.alert()` function identifies events of varying severity levels
+and writes them to the `statuses` measurement in the InfluxDB
+[`_monitoring` system bucket](/influxdb/v2.0/reference/internals/system-buckets/).
+
+_This function is comparable to the [Kapacitor AlertNode](/{{< latest "kapacitor" >}}/nodes/alert_node/)._
+
```
Comment (pierwill) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md:21]: Does this need a heading like `## Example` here?

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+### check
+({{< req >}})
+InfluxDB check data.
+_See [`tickscript.defineCheck()`](/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/definecheck/)._
+
+_**Data type:** Record_
+
+### id
+Function to return the InfluxDB check ID provided by the [`check` record](#check).
```
Comment (pierwill) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md:48]: A thought: `Returns...` might be more readable than `Function to return`.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    crit: (r) => r._value > 30,
+    warn: (r) => r._value > 20,
+    info: (r) => r._value > 10    
+  )
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```javascript
+data = batch
+  |query('SELECT count(value) from errors')
```
Comment (pierwill) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md:125]: Huh, haven't noticed this `|` syntax before...

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/definecheck.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+##### Generate InfluxDB check data
+```javascript
+import "contrib/bonitoo-io/tickscript"
+
+tickscript.defineCheck(
+  id: "000000000000",
+  name: "Example check name",
+)
+
+// Returns: {
```
Comment (pierwill) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/definecheck.md:65]: Maybe something a bit more explicit? `// The script above returns: `.

**Review by pierwill — APPROVED** [review]

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md (hunk 3 above)
Comment (pierwill) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md:125]: Should these be `|>`? There's another one or two in this PR.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md]: Fixed

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md:126]: This is TICKscript, not Flux.

**Pull request merged**